### PR TITLE
Bug fixes (read description) and implementations

### PR DESCRIPTION
**Pull request by kingcreek**
-When a game window such as billiards was opened, and then another pong window was opened and closed, billiards was disconnected since the socket connection was closed.
-The chat did not correctly display the number of unread messages in private chats
-When billiards was being played and the pong window was opened, the billiards canvas moved down through the css that included the canvas and not a specific id
-Added a view in the profile to be able to request the profile of another userid
-From the chat you can now open another user's profile at the request of the subject
-Corrected problem that when having two game instances open, the server received the game inputs from both causing the disconnection by mistake, now it is sent which game the input comes from and the server derives the input received to the corresponding game.

### CODE REVIEW

File: back/core/user_management/views.py [hunk 1]
```diff
@@ -107,6 +107,34 @@ def get(self, request):
...
+            profile_picture_path = user_serializer.data['profile_picture']
+            if profile_picture_path is not None:
+                absolute_profile_picture_path = '/core' + profile_picture_path
+                # Open the profile picture file, read its content, and encode it in base64
+                with open(absolute_profile_picture_path, "rb") as image_file:
+                    encoded_image = base64.b64encode(image_file.read()).decode('utf-8')
+                # Add the base64 encoded image to the serializer data
+                user_data['profile_picture'] = encoded_image
+            
+            if user_serializer.data['TwoFactorAuth'] == True:
```
Comment (RobertoRobrodri) [inline back/core/user_management/views.py:129]: No sé si enviar el QR de otra persona es buena idea

Comment (kingcreek) [reply back/core/user_management/views.py:129]: El qr se almacena ahi?, lo quito jajaja

Comment (kingcreek) [reply back/core/user_management/views.py]: qr eliminado

**Review by RobertoRobrodri — APPROVED** [review]
Increible

**Pull request merged**